keyvault/data-plane/azure-keyvault-test/pom.xml

```diff
@@ -11,11 +11,22 @@
 	</parent>
 	<artifactId>azure-keyvault-test</artifactId>
 	<name>azure-keyvault-test</name>
-	<url>http://maven.apache.org</url>
+	<url>https://github.com/Azure/azure-sdk-for-java</url>
+
+	<distributionManagement>
```

Comment:
Can these properties not be set in the parent pom?

Reply:
I believe I've only included the minimal required set of properties in each pom now. I tried to make it more minimal, but in many cases I found that 'mvn site' would try to be 'smart' and relativize the urls based on the parent/child relationship, leading to URLs that are actually broken. That's why I include more properties than ideal in each child, to prevent this from happening.